python/packages/core/agent_framework/_mcp.py

```diff
@@ -224,13 +224,20 @@ def resolve_type(prop_details: dict[str, Any]) -> type:
         prop_details = json.loads(prop_details) if isinstance(prop_details, str) else prop_details
 
         python_type = resolve_type(prop_details)
+        description = prop_details.get("description", "")
 
         # Create field definition for create_model
         if prop_name in required:
-            field_definitions[prop_name] = (python_type, ...)
+            if description:
```

Comment:
this can be a one liner:

```suggestion
            field_definitions[prop_name] = ((python_type, Field(description=description)) if description else (python_type, ...))
```

Reply:
and maybe even rewrite the logic a bit to create a field and it if has not description or default value then use `...`